doc/changelog.rst

```diff
@@ -1,6 +1,35 @@
 Changelog
 =========
 
+0.5.0 (unreleased)
+--------------------
+
+This HCIPy release provides further speed improvements for Fourier transforms, among some other minor improvements and bugfixes. See the list of all changes below for a summary. This release can now also be installed with Conda. We dropped support for Python 3.6. This version supports Python 3.7+.
+
+List of major changes
+~~~~~~~~~~~~~~~~~~~~~
+
+* Further speed improvements using NumExpr and in-place operations (`@ehpor <https://github.com/ehpor>`__).
+* Autodetect FFTs when doing general Fourier transforms (`@ehpor <https://github.com/ehpor>`__)
+* Added file IO for Field, Grid and ModeBasis (`@ehpor <https://github.com/ehpor>`__)
+* Allow the use of 32bit floating point Wavefronts (`@ehpor <https://github.com/ehpor>`__)
+* Added the LUVOIR-B, GMT, TMT, Hale, HabEx, Hubble, JWST and VLTI telescope apertures (`@ivalaginja <https://github.com/ivalaginja>`__, `@syhaffert <https://github.com/syhaffert>`__ & `@ehpor <https://github.com/ehpor>`__)
+* Improved the speed of spiders and segmented apertures by 4x (`@ehpor <https://github.com/ehpor>`__)
+* Knife edge coronagraph now works in all four directions (`@syhaffert <https://github.com/syhaffert>`__ & `@ehpor <https://github.com/ehpor>`__))
+* Improved support for animations (`@ehpor <https://github.com/ehpor>`__)
+* Added focal-length parameter to Lyot coronagraph (`@syhaffert <https://github.com/syhaffert>`__)
+* Added a telescope pupil introductory tutorial (`@ehpor <https://github.com/ehpor>`__)
+* Allow single-mode fibers to be put at arbitrary positions (`@syhaffert <https://github.com/syhaffert>`__)
```

Comment:
step-index fibers (can have any amount of modes).

Reply:
Fixed.